src/main.rs

```diff
@@ -2,16 +2,38 @@
 
 use std::{fs::{File, self}, io::BufReader};
 
+use bed::Record;
 use noodles_bed as bed;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let src = "../data/bed/well_formed_sample.bed";
+    let src = "data/bed/well_formed_sample.bed";
 
     let mut reader = File::open(src).map(BufReader::new).map(bed::Reader::new)?;
     let reference = fs::read_to_string(src)?;
-    let serialized = bed::to_string(&reader)?;
 
-    assert_eq!(reference, serialized);
+    println!("{:?}", reference);
+
+    let bed_records: Vec<Record<3>> = bed::from_reader(&mut reader);
+    println!("{:?}", bed_records);
+
+    let bed_record_serialization= bed::to_string(&bed_records)?;
+    println!("{:?}", bed_record_serialization);
```

Comment:
The `Debug` version of printing a string escapes characters. If you replace this with the following it should avoid showing the escaped backslashes.
```rust
println!("{}", bed_record_serialization);
```

Reply:
Indeed, there are no escape characters using display:

`chr7    127471196       127472363`

We just have an issue that Vec<Record> does not implement Display, but if we really want it should be easy to implement